assets/js/components/notifications/ErrorNotifications.test.js

```diff
@@ -64,4 +65,21 @@ describe( 'ErrorNotifications', () => {
 			'Site Kit can’t access necessary data'
 		);
 	} );
+
+	it( 'renders `Get help` link', () => {
+		provideUserAuthentication( registry, {
+			unsatisfiedScopes: [
+				'https://www.googleapis.com/auth/analytics.readonly',
+			],
+		} );
+		provideSiteInfo( registry, {
+			setupErrorCode: 'error_code',
+			setupErrorMessage: 'An error occurred',
+		} );
+		const { container } = render( <ErrorNotifications />, {
+			registry,
+		} );
+
+		expect( container ).toHaveTextContent( 'Get help' );
```

Comment:
Perhaps it would be nice to add an assertion to check the Link's URL (errorTroubleshootingLinkURL) as well?